ARM-wvd-templates/HCI/CreateHciHostpoolTemplate.json

```diff
@@ -25,26 +25,26 @@
       "location": {
         "type": "string",
         "metadata": {
-          "description": "The location where the resources will be deployed."
+          "description": "Enter a region where Hostpool, workspace, virtual machines will be created. The metadata for these objects will be stored in the geography associated with the region. Supported regions: eastus and westeurope."
```

Comment:
are those the only 2 regions we support?  will it change.   are we sure we want to put supported regions in the description?